### ISSUE TITLE
Web GUI

### ISSUE THREAD

**Issue by charlesomer** [djmaze/librespot-snapserver#6]
Does this come with the web GUI (Snapweb)? I can't seem to be able to access it at :1780?
`The resource '/' was not found.`

**Comment by charlesomer** [comment]
Also, the Spotify support link in the readme directs to the wrong GitHub page now :)

**Comment by djmaze** [comment]
Snapweb, interesting, didn't even know that existed. Could be a useful addition.

Personally, I don't need it, but I am open for PRs!

The link is fixed, thanks!

**Comment by charlesomer** [comment]
Yeah I think it's supposed to be shipped with Snapcast but I'm guessing the docker install does something differently which means it doesn't automatically get bundled.

**Comment by charlesomer** [comment]
Okay so it looks like Snapweb was only introduced in v0.21 and the version this uses is the default Debian one, which is (I believe) 0.15.0-1 https://packages.debian.org/buster/snapserver.

Maybe it's worth adapting this to use the latest one from the Github repo?

**Comment by charlesomer** [comment]
Hmm maybe not `SNAPCAST_VERSION=0.20.0`, looks like it just needs to be bumped up a version? :)

**Comment by simon-zumbrunnen** [comment]
As far as I know Snapweb was introduced in 0.21.0:

https://github.com/badaix/snapcast/releases/tag/v0.21.0

Is there a reason you are still on 0.19.x?

**Comment by charlesomer** [comment]
Sorry I'm not sure where 0.19.x is coming from, in the makefile it appears the version is: `SNAPCAST_VERSION=0.20.0` so just needs updating by the looks of it?

**Comment by djmaze** [comment]
Yes, I should've tagged 0.20 already. And yes, we can update 0.21. 

The problem is that arm builds are currently broken, which is actually an upstream issue at docker. I hope they fix it soon (or maybe they already fixed it even.)

**Comment by simon-zumbrunnen** [comment]
> Sorry I'm not sure where 0.19.x is coming from, in the makefile it appears the version is: `SNAPCAST_VERSION=0.20.0` so just needs updating by the looks of it?

0.19 is the version packaged for debian bullseye.

> Yes, I should've tagged 0.20 already. And yes, we can update 0.21.
> 
> The problem is that arm builds are currently broken, which is actually an upstream issue at docker. I hope they fix it soon (or maybe they already fixed it even.)

This problem seems to have been around since last summer. Since I don't care about the arm build wouldn't it be possible to release the arm builds with a different version? E.g. use debian bullseye package for arm build (and tag it 0.19) and use the .deb file for the other build?

**Comment by W-Floyd** [comment]
This seems to be available now.
